src/irmin-unix/graphql.ml

```diff
@@ -20,7 +20,10 @@ module Server = struct
       (struct
         let info = Info.v
 
-        let remote = Remote.remote
+        let remote =
+          match Remote.remote with
+          | Some fn -> Some (fun ?headers v -> fn ?headers v)
```

Comment:
is this still needed (the pattern-match looks like a no-op)?

Reply:
It's seems that `Remote.remote` comes from `irmin-git` which expects an `?ctx` argument. The no-op erases the `?ctx` argument which is not used (according to https://github.com/mirage/irmin/pull/1065#discussion_r553243932) by `irmin-graphql`.